src/QMCDrivers/CloneManager.h

```diff
@@ -54,13 +54,17 @@ class CloneManager : public QMCTraits
   ///number of threads
   const IndexType NumThreads;
   ///walkers
```

Comment:
Please add note for XXX_uptr pattern.
Describe XXX_uptr[N-1] and XXX[N] relation.